packages/terra-site/package.json

```diff
@@ -42,7 +42,7 @@
     "terra-application": "0.3.0",
     "terra-arrange": "^0.1.0",
     "terra-button": "^0.1.0",
-    "terra-legacy-theme": "^0.1.0",
+    "terra-legacy-theme": "0.3.0",
```

Comment:
Ah, we should lock down the other terra components to a single version as well. I'll log a PR as we need to do this across all packages. This is to prevent differing versions in use in a bundle.